lib/src/rules_factory.dart

```diff
@@ -1,3 +1,5 @@
+import 'package:dart_code_metrics/src/rules/prefer_on_push_cd_strategy.dart';
```

Comment:
@csepanda in this project, we use relative imports

Reply:
Fixed